README

```diff
@@ -1 +1,67 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Java 8, available at (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+Maven 3+, available at (https://maven.apache.org/download.cgi)
+Python 2.4+, available at (https://www.python.org/download/releases/2.4/)
+
+### Installing
+
+Clone the full distribution from (https://github.com/prestodb/presto-manager)
+
```

Comment:
This line break isn't necessary, and it makes a bit more sense to break only _after_ each step of the instructions (and it makes it look nicer in plain text).